### PR TITLE
feat(server): add version endpoint

### PR DESCRIPTION
**Pull request by nikpivkin**
## Description

Added an endpoint to get the trivy and db versions.

Example output:
```bash
curl -s 0.0.0.0:8080/version | jq
{
  "Version": "dev",
  "VulnerabilityDB": {
    "Version": 2,
    "NextUpdate": "2023-07-25T14:15:29.876639806Z",
    "UpdatedAt": "2023-07-25T08:15:29.876640206Z",
    "DownloadedAt": "2023-07-25T09:36:25.599004Z"
  },
  "JavaDB": {
    "Version": 1,
    "NextUpdate": "2023-07-28T01:03:52.169192565Z",
    "UpdatedAt": "2023-07-25T01:03:52.169192765Z",
    "DownloadedAt": "2023-07-25T09:37:48.906152Z"
  },
  "PolicyBundle": {
    "Digest": "sha256:829832357626da2677955e3b427191212978ba20012b6eaa03229ca28569ae43",
    "DownloadedAt": "2023-07-23T11:40:33.122462Z"
  }
}
```

## Related issues
- Close https://github.com/aquasecurity/trivy/issues/715

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

**Comment by knqyf263** [comment]
>@knqyf263 Have an idea where we can place the version.go file? I have temporarily added this to the utils package.

We may want to add a new package `version`. Then, can we move this variable to the package?
https://github.com/aquasecurity/trivy/blob/07075696d1e60b79b7d19b7224e26ddd187729c7/cmd/trivy/main.go#L17

>Is this a suitable [place in the documentation ](https://aquasecurity.github.io/trivy/v0.43/docs/references/modes/client-server/#server)where we can add information about endpoints?

Yes.

File: pkg/version/version.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	"path/filepath"
+
+	"github.com/aquasecurity/trivy-db/pkg/metadata"
+	javadb "github.com/aquasecurity/trivy-java-db/pkg/db"
+	"github.com/aquasecurity/trivy/pkg/log"
+	"github.com/aquasecurity/trivy/pkg/policy"
+)
+
+var (
+	Version = "dev"
```
Comment (knqyf263) [inline pkg/version/version.go:14]: I want to make it read-only.

```suggestion
	version = "dev"
```

Then, can we add `Version() string` or `AppVersion() string`?

File: pkg/version/version.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	if v.JavaDB != nil {
+		output += formatDbMetadata("Java DB", *v.JavaDB)
+	}
+	if v.PolicyBundle != nil {
+		output += v.PolicyBundle.String()
+	}
+	return output
+}
+
+func BuildVersionInfo(appVersion string, cacheDir string) VersionInfo {
```
Comment (knqyf263) [inline pkg/version/version.go:47]: nit:

```suggestion
func NewVersionInfo(appVersion string, cacheDir string) VersionInfo {
```

Comment (knqyf263) [reply pkg/version/version.go:47]: Also, do we need to pass `appVersion` here? There is a version info in this package?

**Review by AnaisUrlichs — APPROVED** [review]

**Review by knqyf263 — COMMENTED** [review]
You also need to update magefile
https://github.com/aquasecurity/trivy/blob/e1c2a8c8042ec8149c20d0b525f5bf5c3446d2fd/magefiles/magefile.go#L40

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**